packages/nodes-base/nodes/Function/Function.node.ts

```diff
@@ -84,6 +90,48 @@ return items;`,
 			items,
 			// To be able to access data of other items
 			$item: (index: number) => this.getWorkflowDataProxy(index),
+			getBinaryDataAsync: async (item: INodeExecutionData): Promise<IBinaryKeyData | undefined> => {
+				// Fetch Binary Data, if available.
+				if (item?.binary && item?.index !== undefined && item?.index !== null) {
```

Comment:
> item?.index !== undefined && item?.index !== null

Cannot set this to `if (item?.index)`, as `index` may be `0`.

Reply:
This would be better as a code comment than a review comment